nucleus/payara-modules/opentracing-adapter/src/main/java/fish/payara/opentracing/OpenTracingService.java

```diff
@@ -76,10 +78,17 @@
     // The tracer instances
     private static final Map<String, Tracer> tracers = new ConcurrentHashMap<>();
     
+    // The name of the Corba RMI Tracer
+    public static final String PAYARA_CORBA_RMI_TRACER_NAME = "__PAYARA_CORBA_RMI";
+
     @PostConstruct
     void postConstruct() {
         // Listen for events
-        Globals.getDefaultBaseServiceLocator().getService(Events.class).register(this);
+        Events events = getFromServiceHandle(Globals.getDefaultBaseServiceLocator(), Events.class);
+
+        if (events != null) {
+            events.register(this);
+        }
```

Comment:
I wonder what it means where we fail here.
I assume the correct behaviour of this service kind of depends upon receiving the events.
Isn't logging a warning good manners?

Reply:
If events aren't present it means it won't be listening to the internal event bus.
It's only used in the next method, listening for when an application is unloaded so as to remove the tracer for that particular application.

I can add a warning though instead of just carrying on

Reply:
If missing the event is no biggie no warning is needed IMHO.